### DISCUSSION TITLE
plss help  me

### DISCUSSION THREAD

**Discussion by tskgamerz** [microsoft/vscode-cpptools#8979] [Show and tell]
The language server crashed 5 times in the last 3 minutes. It will not be restarted.

**Comment by sean-mcmanus** [comment]
What version of the C/C++ extension are you using? What OS are you on? What are the repro steps? Are you able to get it to not crash, such as from opening a different workspace?

Are you able to attach a debugger to the cpptools process before it crashes to get a call stack?

https://github.com/microsoft/vscode-cpptools/wiki/Attaching-debugger-to-cpptools-or-cpptools%E2%80%90srv
